### PR TITLE
[KODOMAMO-455] Upgrade to support latest tflite_flutter

### PR DESCRIPTION
**Pull request by sarthakydv**

### CODE REVIEW

File: example/audio_classification/lib/classifier.dart [hunk 1]
```diff
@@ -55,17 +53,17 @@ class Classifier {
...
     tensorAudio.loadShortBytes(bytes);
-    final pre = DateTime.now().millisecondsSinceEpoch - pres;
+    // final pre = DateTime.now().millisecondsSinceEpoch - pres;
 
-    final runs = DateTime.now().millisecondsSinceEpoch;
+    // final runs = DateTime.now().millisecondsSinceEpoch;
     interpreter.run(
         tensorAudio.tensorBuffer.getBuffer(), _outputBuffer.getBuffer());
-    final run = DateTime.now().millisecondsSinceEpoch - runs;
+    // final run = DateTime.now().millisecondsSinceEpoch - runs;
```
Comment (abhishek10012001) [inline example/audio_classification/lib/classifier.dart:66]: General observation: you always keep commented lines in your PR. Kindly remove these lines which are not used.

Comment (sarthakydv) [reply example/audio_classification/lib/classifier.dart:66]: Incorrect observation 🗿

Comment (sarthakydv) [reply example/audio_classification/lib/classifier.dart:66]: It is a sample project and these are unused vars, deleted now
image

**Review by abhishek10012001 — APPROVED** [review]

**Review by abhishek10012001 — APPROVED** [review]

File: lib/src/task/bindings/text/qa/bert_qa.dart [hunk 2]
```diff
@@ -10,38 +10,47 @@ import 'package:tflite_flutter_helper/src/task/bindings/dlib.dart';
...
+    Pointer<TfLiteBertQuestionAnswerer>);
 
 // Deletes BertQuestionAnswererQaAnswers instance
-void Function(Pointer<TfLiteQaAnswers>) BertQuestionAnswererQaAnswersDelete = tflitelib
-    .lookup<NativeFunction<_BertQuestionAnswererQaAnswersDelete_native_t>>(
-    'BertQuestionAnswererQaAnswersDelete')
-    .asFunction();
-
-typedef _BertQuestionAnswererQaAnswersDelete_native_t = Void Function(Pointer<TfLiteQaAnswers>);
+void Function(Pointer<TfLiteQaAnswers>) BertQuestionAnswererQaAnswersDelete =
```
Comment (saurabhkumar8112) [inline lib/src/task/bindings/text/qa/bert_qa.dart:49]: workspace level formatting change shouldn't go when you're dealing modifying a part of a package for our use case.
This leads to unnecessary changes. For instance everytime you make some changes to a package to meet your need, you should only touch the parts that need changing

**Review by saurabhkumar8112 — APPROVED** [review]
code lgtm
